help/custom-profile-fields.md

```diff
@@ -85,6 +85,17 @@ checkboxes will be disabled.
 
 {end_tabs}
 
+## Pronouns in @-mentions typeahead suggestions
+When mentioning or addressing a user using the @-mentions feature, the typeahead
+suggestions will automatically include their preferred pronouns. This not only
+personalizes interactions but also promotes a more respectful and inclusive
```

Comment:
I'm in full agreement with the sentiment here, but I don't know that it's necessary to editorialize the _why_ behind this feature in the docs. Others can weigh in on this.

Reply:
Yeah, I think it's better to be more neutral -- this is an option if you want that.

Reply:
Let's drop the documentation changes from this PR, and I'll update the docs as follow-up. I think we can just describe behavior more succinctly in the bulleted list where we list field types, since there's no decision for the admin to make here, except for the unusual case of having more than one pronoun field.

Reply:
I have removed the documentation changes from the pull request. I acknowledge the concerns about the neutrality and conciseness of the documentation, and I believe that these aspects would benefit from the expertise of the experienced.